src/fides/api/schemas/storage/storage.py

```diff
@@ -174,20 +175,21 @@ def validate_details(
 
     @root_validator
     @classmethod
-    def json_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+    def format_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         """
         Custom validation to ensure that local destination formats are valid.
         """
-        json_only_destinations = [StorageType.local.value]
+        restricted_destinations = [StorageType.local.value]
         storage_type = values.get("type")
         response_format = values.get("format")
         if (
-            storage_type in json_only_destinations
+            storage_type in restricted_destinations
             and response_format
-            and response_format != ResponseFormat.json.value
+            and response_format
+            not in [ResponseFormat.json.value, ResponseFormat.html.value]
         ):
             raise ValueError(
-                "Only JSON upload format is supported for local storage destinations."
+                "Only JSON or HTML upload format are supported for local storage destinations."
```

Comment:
Not sure of the historical reason for omitting CSV files from being written to the local storage, so I just gave this a more generic name and allowed the HTML format.